wdl/Tasks04.wdl

```diff
@@ -31,14 +31,14 @@ task SplitVariants {
     set -euo pipefail
     svtk vcf2bed ~{vcf} stdout \
       | awk -v OFS="\t" '(($5=="DEL" || $5=="DUP") && $3-$2>=5000) {print $1, $2, $3, $4, $6, $5}' \
-      | split -l ~{n_per_split} -a 6 - gt5kb.
+      | split --additional-suffix ".bed" -l ~{n_per_split} -a 6 - gt5kb.
     svtk vcf2bed ~{vcf} stdout \
       | awk -v OFS="\t" '(($5=="DEL" || $5=="DUP") && $3-$2<5000) {print $1, $2, $3, $4, $6, $5}' \
-      | split -l ~{n_per_split} -a 6 - lt5kb.
+      | split --additional-suffix ".bed" -l ~{n_per_split} -a 6 - lt5kb.
     if [ ~{generate_bca} == "true" ]; then
       svtk vcf2bed ~{vcf} stdout \
-        | awk -v OFS="\t" '($5!="DEL" && $5!="DUP") {print $1, $2, $3, $4, $6, $5}' \
-        | split -l ~{n_per_split} -a 6 - bca.
+        | awk -v OFS="\t" '($5!="DEL" && $5!="DUP") {print $1, $2RDTestGenotype, $3, $4, $6, $5}' \
```

Comment:
Is the `$2RDTestGenotype` supposed to be there? Looks like it could be a typo?

Reply:
Yes must have accidentally pasted that in somehow